src/app/components/EditorInterface.tsx

```diff
@@ -159,7 +161,7 @@ const EditorInterface = () => {
         editor
           ?.chain()
           .focus()
-          .setContent(originalTemplate as string)
+          .setContent(appState?.appState.originalTemplate as string)
```

Comment:
isn't there a way to use `appState` once @aatbip

Reply:
Yes we can refactor code to use appState only once. This will bring changes to many files and may created breaking changes. I think it is better to do this in a separate branch after we merge this PR. @saugatmaharjan